tests/tests_basics_files.yml

```diff
@@ -74,4 +74,8 @@
 
     - name: Check the test log message in {{ __default_system_log }}
       command: /bin/grep testMessage0000 {{ __default_system_log }}
+      register: __result
+      until: __result.rc == 0
```

Comment:
Use `until: __result is success` instead of comparing `rc` directly - see https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#task-results

Reply:
Thanks, @richm. Done.